app/Helpers/FixometerFile.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace App\Helpers;
```

Comment:
Now that this is namespaced, do we need to update the usages of `FixometerFile`?

Reply:
I didn't see any other errors or warnings. If errors do pop up for "production" then we can address it then.

Reply:
Scratch that, its cause I didn't touch anything revolving image uploads.

We do need to update the usage.

Reply:
Going to follow up on this issue in another pull.